validate/validate.go

```diff
@@ -174,12 +177,15 @@ func byteCheckRtmr(size int, given, required [][]byte) error {
 			return fmt.Errorf("quote field RTMR[%d] is %s. Expect %s",
 				i, hex.EncodeToString(given[i]), hex.EncodeToString(required[i]))
 		}
+		logger.V(2).Infof("Quote field RTMR[%d] matches the input value %v", i+1, given)
```

Comment:
The logger.V(2) logs should be at the start of the function, so that we have debug logs even if there is an unexpected error/early exit/crash.

Reply:
Done